youtube_dl/extractor/bandcamp.py

```diff
@@ -51,11 +75,25 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, title)
         thumbnail = self._html_search_meta('og:image', webpage, default=None)
         m_download = re.search(r'freeDownloadPage: "(.*?)"', webpage)
+        m_trackinfo = re.search(r'trackinfo: (.+),\s*?\n', webpage)
+        json_code = m_trackinfo.group(1) if m_trackinfo else None
+        data = json.loads(json_code)[0] if json_code else None
```

Comment:
Breaks extraction if `json.loads` fails.

Reply:
It was fine when it did the same [in the previous version](https://github.com/rg3/youtube-dl/pull/14060/files/471f398772782ff73542c7fa1490c9ba31eed884#diff-7ca43acb2da36f738f78a676e3ffdafbL58).